scripts/convert_merged_tract_to_dpdd.py

```diff
@@ -230,17 +193,20 @@ def write_dataframe_to_files(
 
 Potential compression algorithms are gzip (default), snappy, lzo, uncompressed.
 Availability depends on the installation of the engine used.
-
-[2018-10-02: The 'dc2_object_run1.2p reader doesn't exist yet.]
-
 """
     parser = ArgumentParser(description=usage,
                             formatter_class=RawTextHelpFormatter)
     parser.add_argument('--tract', type=int, nargs='+', default=[],
                         help='Skymap tract[s] to process.')
-    parser.add_argument('--reader', default='dc2_coadd_run1.1p',
+    parser.add_argument('--reader', default='dc2_object_run1.1p',
                         help='GCR reader to use. (default: %(default)s)')
-    parser.add_argument('--parquet_scheme', default='hive',
+    parser.add_argument('--no-hdf', action='store_false', dest='generate_hdf',
+                        help='disable HDF output')
+    parser.add_argument('--no-fits', action='store_false', dest='generate_fits',
+                        help='disable FITS output')
+    parser.add_argument('--no-parquet', action='store_false', dest='generate_parquet',
+                        help='disable Parquet output')
+    parser.add_argument('--parquet_scheme', default='simple',
```

Comment:
1. I called it `--write_{fits,hdf,parquet}' and I would suggest that be the names of the options here.
2. Add both `--write-*` and `--no-*` command-line options.  Use `%(default)s` in the help string so that the help clearly prints out the default selected. 

E.g.,
```
    parser.add_argument('--write-hdf', default=False, action='store_true',
                        help='Write HDF output.  (default: %(default)s)')
    parser.add_argument('--write-fits', default=False, action='store_true',
                        help='Write FITS output.  (default: %(default)s)')
    parser.add_argument('--write-parquet', default=False, action='store_true',
                        help='Write Parquet output.  (default: %(default)s)')
   parser.add_argument('--no-hdf', action='store_false', dest='write-hdf',
                        help='Disable HDF output.  (default: %(default)s)')
    parser.add_argument('--no-fits', action='store_false, dest='write-fits',
                        help='Distable FITS output.  (default: %(default)s)')
    parser.add_argument('--no-parquet', action='store_false', dest='write-parquet',
                        help='Write Parquet output.  (default: %(default)s)')
```

Reply:
@wmwv I'll be happy to add `--write-*` options but I want to know what's the expected default behavior when no options are given? 

If the default behavior (without any options) is to write out all formats, then I suggest that we change `--write-*` to `--write-*-only` so that these option actually has effects, and we can keep `--no-*` options too. [Note that in this case, the implementation in your example would not work as what we describe here, but I can implement it correctly.]

If the default behavior (without any options) is to write nothing at all, then I suggest that we add the `--write-*` options and also a `--write--all` option, but remove the `--no-*` options.

Reply:
Alternatively, we can get rid of `--write-*` and `--no-*`, and just do this:

```python
parser.add_argument(
    '--format', 
    nargs='+',
    default=['all'],
    choices=['all', 'hdf', 'fits', 'parquet'],
)
```